bundles/org.eclipse.e4.ui.dialogs/src/org/eclipse/e4/ui/dialogs/filteredtree/FilteredTree.java

```diff
@@ -102,21 +102,40 @@ public class FilteredTree extends Composite {
 	 */
 	private static final long SOFT_MAX_EXPAND_TIME = 200;
 
+	/**
+	 * Time for refresh job delay in terms of expansion in long value
+	 */
+	private final long refreshJobDelay;
+
+	/**
+	 * Default time for refresh job delay in ms
+	 */
+	private static final long DEFAULT_REFRESH_TIME = 200;
+
 	/**
 	 * Create a new instance of the receiver.
 	 *
-	 * @param parent
-	 *            the parent <code>Composite</code>
-	 * @param treeStyle
-	 *            the style bits for the <code>Tree</code>
-	 * @param filter
-	 *            the filter to be used
-	 */
-	public FilteredTree(Composite parent, int treeStyle, PatternFilter filter) {
+	 * @param parent               the parent <code>Composite</code>
+	 * @param treeStyle            the style bits for the <code>Tree</code>
+	 * @param filter               the filter to be used
+	 * @param refreshDelayInMillis refresh delay in ms, the time to expand the tree
+	 *                             after debounce
+	 * @since 1.4
```

Comment:
This tag is not correct, is it? In 1.4.0, this constructor did not exist. Shouldn't the minor version of the bundle be incremented when adding a constructor, i.e., new public API?

Reply:
@HeikoKlare should the minor version be 5 for this? If yes, then who update it in Manifest.mf file?

Reply:
Yes, I would expect the minor version to be incremented to 5. You can simply replace the version `1.4.300.qualifier` in the Manifest with `1.5.0.qualifier`